docs/reference/how-to/indexing-speed.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[tune-for-indexing-speed]]
+== Tune for indexing speed
```

Comment:
I think it is tricky, isn't it, it all depends what is currently hurting your indexing speed most, which might be I/O is some cases and CPU in other cases?